llvm/include/llvm/Analysis/CmpInstAnalysis.h

```diff
@@ -102,17 +102,21 @@ namespace llvm {
   };
 
   /// Decompose an icmp into the form ((X & Mask) pred C) if possible.
-  /// Unless \p AllowNonZeroC is true, C will always be 0.
+  /// Unless \p AllowNonZeroC is true, C will always be 0. If \p
+  /// DecomposeAnd is specified, then, for equality predicates, this will
+  /// decompose bitmasking (e.g. implemented via `and`).
```

Comment:
```suggestion
  /// decompose bitmasking via `and`.
```
Don't think "e.g." makes sense with the new option name.